amundsen_application/static/js/components/OwnerEditor/styles.scss

```diff
@@ -3,25 +3,10 @@
 @import 'variables';
 
 .owner-editor-component {
-  .btn.add-list-item {
-    color: $gray-light;
+  .btn.add-item-button {
     padding: 4px 1px;
```

Comment:
I the padding and height should get baked into the `btn-icon` class, perhaps with a second modifier that indicates it comes with text.

Reply:
So you're suggesting a new style, or modifying the existing one to apply the relevant styles to text if it exists?